mpv-discordRPC/main.lua

```diff
@@ -277,4 +282,5 @@ mp.add_key_binding(o.key_toggle, "active-toggle", function()
 	{repeatable=false})
 
 -- run `main` function
+io.popen('python3 "/media/isaac/Games/Home/isaac/.config/mpv/scripts/mpv-discordRPC/python-pypresence.py" &')
```

Comment:
Please don't use hard coded file path.
just use `python` instead of `python3` explicitly

Reply:
When I used `python` instead of `python3` it gave me errors about pypresense not existing, and when I installed it with pip it gave me invalid syntax errors. I guess that could just be a Linux thing